Common/src/CConfig.cpp

```diff
@@ -5875,7 +5889,7 @@ void CConfig::SetOutput(SU2_COMPONENT val_software, unsigned short val_izone) {
           case SA_ZDES:  cout << "Delayed Detached Eddy Simulation (DDES) with Vorticity-based SGS" << endl; break;
           case SA_EDDES: cout << "Delayed Detached Eddy Simulation (DDES) with Shear-layer Adapted SGS" << endl; break;
         }
-        if (using_uq){
+        if (sstParsedOptions.production == SST_OPTIONS::UNCERTAINTY){
```

Comment:
Why did these change?  Is using_uq not set properly before hand?

Reply:
I'm trying to get all regression tests running again, but the results are different between my machine and github, so that's why I pushed this intermediate solution to see what the differences are. I will fix it this weekend and make sure that the uq+sust+sst regressions are working with the sstparsedoptions implementation.